pmoves/Makefile

```
@@ -394,6 +409,48 @@ SUPABASE_DB_HELPER ?= scripts/supabase/find_db_container.sh
 up-supabase: ## Start Supabase using selected runtime (SUPABASE_RUNTIME=cli|kong|compose)
 	@$(MAKE) --no-print-directory supa-start
 
+.PHONY: kong-routes-check
+KONG_ADMIN_URL ?= http://localhost:8001
+KONG_CONTAINER ?= pmoves-supabase-kong-1
+kong-routes-check: ## Assert the Kong gateway is actually WIRED: routes loaded + every upstream resolvable
```

checkmake (0.3.2)

[warning] Target body for "kong-routes-check" exceeds allowed length of 5 lines (37).

(maxbodylength)